src/pages/billing/Project.js

```diff
@@ -368,6 +381,19 @@ const ProjectDetail = ({ billingProject, reloadBillingProject, billingAccounts,
             }
           }
         }, [icon('edit', { size: 12 })]),
+        h(Link, {
+          tooltip: 'Remove Billing Account',
+          style: { marginLeft: '0.5rem' },
+          baseColor: colors.danger,
+          onClick: async () => {
+            if (Auth.hasBillingScope()) {
+              setShowBillingRemovalModal(true)
+            } else {
+              await authorizeAndLoadAccounts()
+              setShowBillingRemovalModal(Auth.hasBillingScope())
```

Comment:
the remove button is still clickable when there is no billing account. Maybe we should disable or hide the button in this case?

![Screen Shot 2021-11-01 at 12 26 22 PM](https://user-images.githubusercontent.com/5375000/139706162-6e1a2d21-05e7-41be-9634-df0b6a0851e5.png)